File: 03-conditionals/if-else.js
let age = 20;

if(age >= 18){
    console.log("Adult");
}
else if(age>= 13){
    console.log("Teen");
}

else {
    console.log("child")
}

// Truthy/Falsy: JavaScript evaluates conditions based on truthy (e.g., non-zero numbers, non-empty strings) and falsy values (0, "", null, undefined, false, NaN).

let value = ""

if( !value ) console.log("false value")


// Nested conditions

hasLicense = true

if( age>= 18){
    if(hasLicense){
        console.log("You can drive!")
    }
    else{
        console.log("Adult but can't drive")
    }
}
else{
    console.log("Too young to drive")
}

File: 03-conditionals/if_else.py
age =20

if age>=18:
    print("Adult")
elif age>=13:
    print("Teen")
else:
    print("Child")

# Truthy/Falsy: Python also uses truthy/falsy evaluation, but its falsy values are slightly different (0, 0.0, "", [], {}, None, False).

value = []

if not value:
    print("False value")


# Nested conditions

has_license = True

if age >=20:
    if has_license:
        print("Yes you can drive")
    else:
        print("Adult but can't drive")
else:
    print("Too young to drive")